### DISCUSSION TITLE
Życzenia świąteczne 2021/2022

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#20575] [General] [answered]
Idą święta, zaproś gości.
Wydłub z karpia wszystkie ości.
Powyjadaj z barszczu uszy.
A gdy Cię za bardzo suszy,
Przysiądź z buteleczką wina,
I obejrzyj dziś Kevina!

**Comment by blocker999** [comment]
Wesołych i spokojnych Świąt Bożego Narodzenia oraz wystrzałowego Sylwestra i spełnienia najskrytszych marzeń w Nowym Roku

**Comment by hawkeye116477** [comment, marked as answer]
Niechaj w Nowym Roku ceny paliw będą niskie,
niech Reprezentacja Polski w nogę dobrze ciśnie,
niechaj wiek emerytalny pójdzie w dół, a płace w górę,
i żeby nowe Gwiezdne Wojny były jak "Imperium Kontratakuje"
Tego wam życzę w Nowym Roku 2022.
